packages/web/config/price.ts

```diff
@@ -1355,10 +1355,20 @@ export const PoolPriceRoutes: IntermediateRoute[] = [
     alternativeCoinId: "pool:fox",
     poolId: "949",
     spotPriceSourceDenom: DenomHelper.ibcDenom(
-        [{ portId: "transfer", channelId: "channel-169" }],
-        "cw20:juno1u8cr3hcjvfkzxcaacv9q75uw9hwjmn8pucc93pmy6yvkzz79kh3qncca8x"
+      [{ portId: "transfer", channelId: "channel-169" }],
+      "cw20:juno1u8cr3hcjvfkzxcaacv9q75uw9hwjmn8pucc93pmy6yvkzz79kh3qncca8x"
     ),
     spotPriceDestDenom: "uosmo",
     destCoinId: "pool:uosmo",
   },
+  {
+    alternativeCoinId: "pool:unls",
+    poolId: "781",
+    spotPriceSourceDenom: DenomHelper.ibcDenom(
+      [{ portId: "transfer", channelId: "channel-1837" }],
+      "unls"
+    ),
+    spotPriceDestDenom: "aUSDC",
+    destCoinId: "pool:aUSDC",
```

Comment:
The coin id hasn't been defined, yet, so we could probably just use the coingecko price?
`destCoinId: "usd-coin"`

Reply:
Fixed